01-node-tutorial/answers/09-path-module.js

```diff
@@ -0,0 +1,3 @@
+const path = require('path')
+
+console.log(path.join(__dirname))
```

Comment:
We want path.join to join multiple strings, what the different parts of path that can be joined together?

Reply:
Thanks, I made the changes.